src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -26,13 +26,15 @@
     private final TranscriptParser transcriptParser;
     private final AddressBookParser addressBookParser;
 
+    //@@author alexkmj
     public LogicManager(Model model) {
         this.model = model;
         history = new CommandHistory();
         transcriptParser = new TranscriptParser();
         addressBookParser = new AddressBookParser();
     }
 
+    //@@author alexkmj
```

Comment:
I think as long as the previous @@author comment does not see any new @@author tag it'll continue to count the rest below as yours. But yea trivial.

Reply:
I added individually because of `getFilteredPersonList` etc